core-web/libs/dotcms-scss/angular/dotcms-theme/components/_dialog.scss

```diff
@@ -106,3 +106,9 @@
     -webkit-backdrop-filter: blur($blur-md);
     backdrop-filter: blur($blur-md);
 }
+
+.p-dialog-mask.p-dialog-mask-transparent-nested.p-component-overlay {
+    background-color: transparent;
+    -webkit-backdrop-filter: none;
+    backdrop-filter: none;
+}
```

Comment:
Why?

Reply:
When we open a nested primeng dialog, the background turns black
This css avoid that behavoiur